c2corg_ui/templates/whatsnew.html

```diff
@@ -0,0 +1,18 @@
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
```

Comment:
On the other hand the HTML page title is not correct (as for now is is "home page") => please update the page title.